app/components/notification-dropdown.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
```

Comment:
<img width="905" alt="screen shot 2017-05-19 at 11 23 42 pm" src="https://cloud.githubusercontent.com/assets/2404372/26260340/734c5bd4-3cea-11e7-948f-9e03d1b8a12e.png">

- Why is there so much space everywhere ? 
- Why different widths on desktop ? Keep the same width as on mobile. (See how facebook does it) 
- Mark all read button seems to be mis-aligned and it looks the same as rest of the text. 
- See current implementation at https://eventyay.com

Have added more comments inline. 

Overall the UI looks _very_ unrefined.